code/src/GridAPI/Sort.ts

```diff
@@ -8,41 +8,95 @@ namespace GridAPI.Sort {
      *
      * @export
      * @param {string} gridID ID of the Grid that is to be to check from results.
-     * @returns {*}  {void}
+     * @returns {*}  {string} Return Message containing the resulting code from sorting columns and the error message in case of failure
      */
-    export function Clear(gridID: string): void {
+    export function Clear(gridID: string): string {
         PerformanceAPI.SetMark('Sort.Clear');
 
-        if (!OSFramework.Helper.IsGridReady(gridID)) return;
-        const grid = GridManager.GetGridById(gridID);
+        let returnMessage = {
+            isSuccess: true,
+            message: 'Success',
+            code: OSFramework.Enum.ErrorCodes.GRID_SUCCESS
+        };
 
-        grid.features.sort.clear();
+        if (!OSFramework.Helper.IsGridReady(gridID)) {
+            returnMessage = {
+                isSuccess: false,
+                message: 'Grid not found',
```

Comment:
This text should not be hardcoded here. We should have an Enum `ErrorMessages`.

Reply:
I already talked about that we @glundgren93 since we have a lot of error messages all over the place

Reply:
This is done @rugoncalves!